pkgs/by-name/co/code-cursor/package.nix

```
@@ -102,8 +144,11 @@ stdenvNoCC.mkDerivation {
     changelog = "https://cursor.com/changelog";
     license = lib.licenses.unfree;
     sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
-    maintainers = with lib.maintainers; [ sarahec ];
-    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [
+      sarahec
+      aspauldingcode
+    ];
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

```suggestion
    platforms = supportedPlatforms;
```

Since we have an explicit list, let's use it.